src/components/uac/examples/uac-error/index.njk

```diff
@@ -0,0 +1,64 @@
+---
+layout: none
+---
+{% extends "styles/page-template/_template.njk" %}
+{% from "components/uac/_macro.njk" import onsUACInput %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+{% set pageConfig = {
+  "title": "Access Code example"
+} %}
+
+{% block main %}
+    <h1 class="u-mt-l">Start survey</h1>
+
+    {% call
+        onsPanel({
+            title: "There is a problem with your answer",
+            type: "error"
+        })
+    %}
+    <p>These <strong>must be corrected</strong> to continue.</p>
+    {% from "components/lists/_macro.njk" import onsList %}
+    {{
+        onsList({
+            "classes": "list--bare",
+            "itemsList": [
+                {
+                    "text": "Enter an access code to continue",
+                    "index": true,
+                    "url": "#uac",
+                    "classes": "js-inpagelink"
+                }
+            ]
+        })
+    }}
+    {% endcall %}
+
+    {{ onsUACInput({
```

Comment:
All text inputs within error panels should have a red border until focussed... 
<img width="704" alt="Screenshot 2020-06-16 at 15 05 17" src="https://user-images.githubusercontent.com/43346934/84785044-e81e7000-afe2-11ea-86af-7cae535e0265.png">